### PR TITLE
dumpsys: refresh page: standardization

### PR DESCRIPTION
**Pull request by EmilySeville7cfg**
- use `display` term
- add mnemonics
- use range placeholder for seconds
- suggest standardardization for default values

- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/android/dumpsys.md [hunk 1]
```diff
@@ -24,6 +24,6 @@
 
 `dumpsys --skip {{service}}`
 
-- Specify a timeout period in seconds (defaults to 10s):
+- Specify a [t]imeout period in seconds (`-t` default: 10s):
 
-`dumpsys -t {{seconds}}`
+`dumpsys -t {{1..infinity}}`
```
Comment (kbdharun) [inline pages/android/dumpsys.md:29]: I am not sure about this placeholder. Let's see what others think.

**Review by kbdharun — APPROVED** [review]
l, t doesn't have long options. But I guess it is acceptable due to the proposed mnemonic rule.

File: pages/android/dumpsys.md (hunk 1 above)
Comment (EmilySeville7cfg) [reply pages/android/dumpsys.md:29]: I am not sure just about the range of possible values of `-t`. ;)

**Comment by EmilySeville7cfg** [comment]
This PR can be closed: #9716 is a replacement for it.

**Pull request closed without merging**